### PR TITLE
Bug 11694: various errors in GenericTicketConnectorSOAP.wsdl

### PR DESCRIPTION
**Pull request by StCyr**
SLA, SLAID, Service, and ServiceID are not mandatory; They are not returned by OTRS when not using "Services"

### CODE REVIEW

**Comment by mgruner** [comment]
Hello @StCyr, this does not merge cleanly, can you please rebase your branch?

**Comment by StCyr** [comment]
I don't know against wich branch I should rebase; I'll try with master

**Comment by StCyr** [comment]
Sorry for some reason I can't rebase my branch.

I've thus created another one. The corresponding PR is there: https://github.com/OTRS/otrs/pull/847

You can close this PR

File: development/webservices/GenericTicketConnectorSOAP.wsdl [hunk 1]
```diff
@@ -1035,7 +1035,7 @@
                     </xsd:element>
                     <xsd:element
                         name="UntilTime"
-                        type="xsd:positiveInteger"
+                        type="xsd:nonNegativeInteger"
```
Comment (carlosfrodriguez) [inline development/webservices/GenericTicketConnectorSOAP.wsdl]: Hi @StCyr 

Thanks you for this pull request, can you tell me why should we change the positiveInteger to nonNegativeInterger? what would be the advantages?

((enjoy))
cr

**Pull request closed without merging**